grizli/prep.py

```diff
@@ -4716,14 +4932,18 @@ def visit_grism_sky(grism={}, apply=True, column_average=True, verbose=True, ext
                         label=grism['files'][j].split('_fl')[0])
 
         # result
-        fp = open(file.replace('_flt.fits', '_column.dat'), 'wb')
+        fp = open(file.replace('_flt.fits', '_column.dat').replace('_rate.fits', '_column.dat'), 'wb')
```

Comment:
Are we sure that the JWST grism exposures are being rotated correctly in this `prep` processing so that the column subtraction is truly columns parallel to the dispersion axis as expected?

Reply:
I put some functions to compute the rotations in [grismconf.py](https://github.com/gbrammer/grizli/blob/master/grizli/grismconf.py#L590)